applications/jonny5/src/j5_limits.erl

```diff
@@ -64,10 +67,12 @@
 -spec get(ne_binary()) -> limits().
 get(Account) ->
     AccountId = wh_util:format_account_id(Account, 'raw'),
-    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
-        {'ok', Limits} -> Limits;
+    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
+        {'ok', _Limits} -> _Limits;
```

Comment:
Looks like commit cb1115fd6343f299e015f4ff3bae2b28ffd25a41 shows the diff the way it should.
I filed a ticket with github.